app-emulation/retroarch/metadata.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>mjaoune55@gmail.com</email>
+		<name>Mahmoud Jaoune</name>
+	</maintainer>
+	<longdescription lang="en">
+	RetroArch is a frontend for emulators, game engines and media players.
+
```

Comment:
I think those extra lines can be removed. Also, indenting it would be great (check for example www-client/uget).